packages/mongodb-constants/README.md

```diff
@@ -0,0 +1,21 @@
+# get-os-info
```

Comment:
Maybe need to update readme?

Reply:
Whoops, thanks for catching!

Reply:
Fixed in https://github.com/mongodb-js/devtools-shared/pull/34/commits/a41288c83fc781551b27bc8bd257687a8cd12f07